### ISSUE TITLE
Azure CLI replaces VSTS CLI

### ISSUE THREAD

**Issue by pinkfloydx33** [dahlbyk/posh-git#695]
The VSTS CLI has been deprecated and replaced with the Azure CLI (`az repos`). 

The aliases that are added via [`--use-git-aliases`](https://docs.microsoft.com/en-us/azure/devops/cli/auto-detect-and-git-aliases?view=azure-devops) vary slightly from those added by the VSTS CLI (`pr` vs `vsts.pr`). Additionally, some of the [sub-commands](https://docs.microsoft.com/en-us/cli/azure/ext/azure-devops/repos/pr?view=azure-cli-latest) have changed; many of them are pluralization (`reviewer` vs `reviewers`) but some options have been removed and a few added.

Any chance the module will be updated to officially support the aliases added by the Azure CLI?

I've updated my local copy of the module and would gladly submit a PR if you'd like.

**Comment by dahlbyk** [comment]
> I've updated my local copy of the module and would gladly submit a PR if you'd like.

That would be great!

Is it reasonable to retain support for `vsts` and add parallel support for `az devops` aliases?

**Comment by pinkfloydx33** [comment]
Yes we can support both aliases. I had to make a few tweaks to my local after you said that, but its easily done. The alias contents are different so we can key off KnownAliases and it works fine. You'll never have both installed, but if you did for some reason they shouldn't conflict.

I also added support for those commands that have sub-commands; for example `git pr policy list - -params` with policy having sub-commands (list, etc). There are a few others: reviewers and work-items. I did not back-port that to the Vsts version however, so azure cli only.

I'll have a PR likely later today. I've also added support for #691. Should I submit two seperate PRs?

**Comment by dahlbyk** [comment]
> Should I submit two seperate PRs?

Definitely separate commits, but a single PR is fine.
